lib/puma/app/status.rb

```diff
@@ -16,47 +16,51 @@ def initialize(cli, token = nil)
 
       def call(env)
         unless authenticate(env)
-          return rack_response(403, 'Invalid auth token', 'text/plain')
+          return rack_response 403, 'Invalid auth token', 'text/plain'
         end
 
         case env['PATH_INFO']
         when /\/stop$/
           @cli.stop
-          rack_response(200, OK_STATUS)
+          rack_response 200, OK_STATUS
 
         when /\/halt$/
           @cli.halt
-          rack_response(200, OK_STATUS)
+          rack_response 200, OK_STATUS
 
         when /\/restart$/
           @cli.restart
-          rack_response(200, OK_STATUS)
+          rack_response 200, OK_STATUS
 
         when /\/phased-restart$/
           if !@cli.phased_restart
-            rack_response(404, '{ "error": "phased restart not available" }')
+            rack_response 404, '{ "error": "phased restart not available" }'
           else
-            rack_response(200, OK_STATUS)
+            rack_response 200, OK_STATUS
           end
 
         when /\/reload-worker-directory$/
           if !@cli.send(:reload_worker_directory)
-            rack_response(404, '{ "error": "reload_worker_directory not available" }')
+            rack_response 404, '{ "error": "reload_worker_directory not available" }'
           else
-            rack_response(200, OK_STATUS)
+            rack_response 200, OK_STATUS
```

Comment:
If we're gonna do this (also changing to single-quotes), we should also be added Rubocop rules.

Reply:
BTW, I didn't change all of them (currently in master), note both single & double quotes.  So I just made them consistent...

https://github.com/puma/puma/blob/1d97e42dde2bcc737f8d01f6c2292f6845c9c504/lib/puma/app/status.rb#L59